evaluate/task/symflower.go

```diff
@@ -31,6 +32,29 @@ func symflowerFix(logger *log.Logger, repositoryPath string, language language.L
 	return uint64(time.Since(start).Milliseconds()), nil
 }
 
+// symflowerTemplate runs the "symflower utg" command and returns its execution time in milliseconds.
+func symflowerTemplate(logger *log.Logger, repositoryPath string, language language.Language, filePath string) (duration uint64, err error) {
+	start := time.Now()
+
+	o, err := util.CommandWithResult(context.Background(), logger, &util.Command{
+		Command: []string{
+			tools.SymflowerPath, "uts",
+			"--language", language.ID(),
+			"--workspace", repositoryPath,
+			filePath,
+		},
+
+		Directory: repositoryPath,
+	})
+	if err != nil {
+		return 0, pkgerrors.WithStack(err)
+	}
+
+	fmt.Println(o)
```

Comment:
Use logging instead

Reply:
woops. leftover debugging

Reply:
Think the command output is already in the log since the logger is passed to the `exec`.